### PR TITLE
revome methods from labels factory

### PR DESCRIPTION
**Pull request by shubham808**
[WIP] its a start towards eventually deleting CLabelsFactory.

### CODE REVIEW

File: src/shogun/labels/LabelsFactory.h [hunk 1]
```diff
@@ -29,29 +29,6 @@ namespace shogun
 class CLabelsFactory : public CSGObject
 {
 public:
-	/** specialize a base class instance to CBinaryLabels
```
Comment (karlnapf) [inline src/shogun/labels/LabelsFactory.h]: i guess you can just delete the file?

**Comment by shubham808** [comment]
@karlnapf see [here](https://gist.github.com/shubham808/864261eed299b5b1a8fbcb89d6d260e9) there is something wrong.
However the KNN unit test works fine.. thats a good sign :)

File: src/shogun/labels/LabelsFactory.h (hunk 1 above)
Comment (shubham808) [reply src/shogun/labels/LabelsFactory.h]: yes i am working on it

**Comment by karlnapf** [comment]
You will need to run this before and after the changes, see if there is a difference.
If there is, pls identify the change that causes the difference and point me to it. Then I can help

**Comment by shubham808** [comment]
I dont think it is related to my changes (StanPerceptron error)

**Comment by shubham808** [comment]
@karlnapf same thing happened with valgrind on develop too so it is probably not related to this pr

**Comment by karlnapf** [comment]
Cool then!

**Pull request merged**